autoload/ctrlsf/win.vim

```diff
@@ -70,8 +70,13 @@ func! ctrlsf#win#OpenMainWindow() abort
               \ 'bottom' : 'botright', 'right' : 'botright vertical'}
               \[g:ctrlsf_position] . ' '
     else
-        " compact mode: fixed window size and position
-        let winsize = 10
+        if g:ctrlsf_compact_winsize =~ '\d\{1,2}%'
+          let winsize = &lines * str2nr(g:ctrlsf_winsize) / 100
```

Comment:
I think using `ctrlsf_compact_winsize` instead of `ctrlsf_winsize` makes more sense maybe.

Reply:
woops, of course!